examples/pymc3_howto/lasso_block_update.ipynb

```diff
@@ -4,40 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<p>pylab should not be used anymore as a general rule, pyplot can be used in its place. However, in this particular case, I think it's better to use <code>plot_pair</code> , see example in https://arviz-devs.github.io/arviz/examples/plot_joint.html (it uses plot_pair, not plot_joint). Feel free to set marginals to True or to False here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>